### DISCUSSION TITLE
priority request mysql.createPool

### DISCUSSION THREAD

**Discussion by FridayJew** [sidorares/node-mysql2#1487] [Q&A]
How i can run priority request to db in mysql.createPool if all pool are full ? My script can update db for 1-2 minutes, in this time all other request waiting end of this.
I can make two mysqlpool, maybe its working, but i want found another way for this, if it posible.

**Comment by sidorares** [comment]
I think having separate pool that is only used for high priority queries is probably the best option right now ( or alternatively just create a new single connection and discard it after priority query is finished )
